core/src/test/java/jenkins/model/RunIdMigratorTest.java

```diff
@@ -83,13 +83,13 @@
 
     @Test public void legacy() throws Exception {
         assumeFalse("Symlinks don't work well on Windows", Functions.isWindows());
-        write("2014-01-02_03-04-05/build.xml", "<?xml version='1.0' encoding='UTF-8'?>\n<run>\n  <stuff>ok</stuff>\n  <number>99</number>\n  <otherstuff>ok</otherstuff>\n</run>");
+        write("2014-01-02_03-04-05/build.xml", "<?xml version='1.1' encoding='UTF-8'?>\n<run>\n  <stuff>ok</stuff>\n  <number>99</number>\n  <otherstuff>ok</otherstuff>\n</run>");
```

Comment:
so I think these changes are wrong - they are testing you can migrate from an old build layout to the new layout - as such the files would have been written with the old XStream utilities and be `xml 1.0`

Reply:
but when they get migrated they will be written out as xml 1.1.  I'll take a closer look and see what i can do about that.

Reply:
After looking into what the migrator does, there are no XStream operations involved, so xml version will not be changed.  Reverting these changes